src/app/clusters/color-control-server/color-control-server.cpp

```diff
@@ -84,6 +84,18 @@ enum
     TEMPERATURE_TO_TEMPERATURE = 0x22
 };
 
+enum
```

Comment:
Do we need those two enums? Booleans should be fine
isColorLoopActive = true/false
IsColorLoopDirection =true/false